test/e2e/test_broker_pubsub.go

```diff
@@ -46,119 +49,102 @@ PubSubWithBrokerTestImpl tests the following scenario:
 Note: the number denotes the sequence of the event that flows in this test case.
 */
 
-func BrokerWithPubSubChannelTestImpl(t *testing.T, packages map[string]string) {
+func BrokerWithPubSubChannelTestImpl(t *testing.T) {
 	brokerName := helpers.AppendRandomString("pubsub")
 	dummyTriggerName := "dummy-broker-" + brokerName
 	respTriggerName := "resp-broker-" + brokerName
 	kserviceName := helpers.AppendRandomString("kservice")
 	senderName := helpers.AppendRandomString("sender")
 	targetName := helpers.AppendRandomString("target")
-	clusterRoleName := helpers.AppendRandomString("e2e-pubsub")
-
-	client := Setup(t, true)
-	defer TearDown(client)
-
-	config := map[string]string{
-		"namespace":        client.Namespace,
-		"brokerName":       brokerName,
-		"dummyTriggerName": dummyTriggerName,
-		"respTriggerName":  respTriggerName,
-		"kserviceName":     kserviceName,
-		"senderName":       senderName,
-		"targetName":       targetName,
-		"clusterRoleName":  clusterRoleName,
-	}
-	for k, v := range packages {
-		config[k] = v
-	}
 
-	// Create resources.
-	brokerInstaller := createResource(client, config, []string{"pubsub_broker", "istio"}, t)
-	defer deleteResource(brokerInstaller, t)
+	client := lib.Setup(t, true)
+	defer lib.TearDown(client)
+
+	// Create a new Broker.
+	// TODO(chizhg): maybe we don't need to create these RBAC resources as they will now be automatically created?
+	client.Core.CreateRBACResourcesForBrokers()
+	client.Core.CreateBrokerOrFail(brokerName, lib.ChannelTypeMeta)
+
+	// Create a Trigger with the Knative Service subscriber.
+	client.Core.CreateTriggerOrFail(
+		dummyTriggerName,
+		eventingtestresources.WithBroker(brokerName),
+		eventingtestresources.WithAttributesTriggerFilter(
+			"", "",
```

Comment:
what does "", "" refers to? Can you add `/* <description> */` for non-IDE reference...

Reply:
Changed to `eventingv1alpha1.TriggerAnyFilter`